docs/configuration.rst

```diff
@@ -12,28 +12,31 @@ If you wish to pass extra flags (e.g. :code:`--ignore W503` to :code:`flake8`) y
 
     nbqa flake8 my_notebook.ipynb --ignore W503
 
-or you can put the following in your :code:`setup.cfg` file
+or you can put the following in your :code:`pyproject.toml` file
 
-.. code-block:: cfg
+.. code-block:: toml
+
+    [tool.nbqa.addopts]
+    flake8 = [
+        "--ignore=W503"
+    ]
 
-    [nbqa.addopts]
-    flake8 = --ignore W503
 
 Config file
 ~~~~~~~~~~~
 
-By default, :code:`nbQA` will use :code:`setup.cfg` to get the configs for your third-party tool.
+By default, :code:`nbQA` will look up the configuration of the third party tool in the following files :code:`setup.cfg`, :code:`tox.ini` and :code:`pyproject.toml`.
```

Comment:
I think we can just mention `pyproject.toml` here, if it's now a required dependency

Reply:
nevermind me, this comment is correct, sorry :innocent: